tox.ini

```diff
@@ -5,10 +5,7 @@ envlist = black,flake8,py39
 deps =
     -rrequirements.txt
     -rrequirements-web.txt
-    jsonschema
-    pytest
-    pytest-cov
-    pyyaml
+    -rrequirements-test.txt
```

Comment:
_I think we don't need to pin the testing dependencies. Pinning them all in one file has the disadvantage that we would now need to install a bunch of unrelated packages just to run flake8 or black_
@chmeliik We must provide hashes for all dependencies in case we provided it for at least one dependency. (in `rrequirements.txt`)

Reply:
Ah, that makes sense